docs/modules/ROOT/pages/getting-started/configuration.adoc

```diff
@@ -0,0 +1,257 @@
+= Configuring KubeArchive
+
+This document explains how to configure KubeArchive so you can archive 
+resources and query them.
+
+== Prerequisites
+* KubeArchive installed and running in a cluster (xref:getting-started/installation.adoc[])
+* Know how to query the KubeArchive API (xref:getting-started/kubearchive-api.adoc[])
+
+== The KubeArchiveConfig resource
+
+To configure KubeArchive to archive or delete resources create a
+KubeArchiveConfig custom resource. KubeArchiveConfigs are limited to 1 per namespace
+and should be named `kubearchive`. KubeArchiveConfigs have this general form:
+
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources: [...]
+----
+
+`spec.resources` is a list of elements, each defining rules for a specific kind, so KubeArchive knows
+when to archive or delete them.
+
+==  `selector`: Selecting Resources
+
+Resources in the KubeArchiveConfig are defined using a `selector` which
+requires two keys: `kind` and `apiVersion`. Each entry on `spec.resources`
+requires a `selector`:
+
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources:
+    - selector:
+        apiVersion: v1
+        kind: Pod
+      ...
+    - selector:
+        apiVersion: batch/v1
+        kind: Job
+      ...
+    - selector:
+        apiVersion: apps/v1
+        kind: Deployment
+      ...
+----
+
+[TIP]
+====
+The `kind` provided in a selector must be singular. For example `kind: Pod` is valid, but `kind: Pods` is invalid.
```

Comment:
If we specify this we probably want to clarify also that the word should be capitalized as it is based on the `kind` property of the resource definition and not on the `singular` or `plural` name properties.

```suggestion
The `kind` property in a selector should not be confused with the `singular` or `plural` properties of the resource. 
It should be in singular and capitalized. 
For example `kind: Pod` is valid, but `kind: Pods` is invalid.
```

Reply:
Hmm.. I think we are getting confusing with this block. I fundamentally agree with the changes you propose Marta, and after reading them I also think this block shouldn't exist. `kind` is `kind` and I feel that specifying what it or it isn't makes it more confusing. I will remove this block and if users start wondering what `kind` is, then we can bring it back.